test/unit/model/test_train.py

```diff
@@ -4,16 +4,25 @@
 
 from datetime import datetime, timedelta
 from test.utils.data import TestData
-from unittest.mock import patch
+from unittest.mock import patch, MagicMock
 import unittest
+from plotly import graph_objects as go
 
 import numpy as np
 import pandas as pd
 
 from openstf.model.general import MLModelType, split_data_train_validation_test
 from openstf.model.serializer.creator import ModelSerializerCreator
 from openstf.model.serializer.xgboost.xgboost import XGBModelSerializer
-from openstf.model.train import is_data_sufficient
+from openstf.model.train import (
+    is_data_sufficient,
```

Comment:
you could just import `train` as well here -> `from openstf.model import train` this would give you a shorter import and it would be very clear which functions belong to the `train` module since you would have to call then with the module name -> `train.create_model`, `train.preprocess` etc.

Reply:
thanks for the tip